frontend/packages/console-shared/src/utils/storage-utils.ts

```diff
@@ -0,0 +1,19 @@
+import * as _ from 'lodash';
+import { StorageClass } from '@console/internal/components/storage-class-form';
+
+export const cephStorageProvisioners = [
+  'ceph.rook.io/block',
+  'cephfs.csi.ceph.com',
+  'rbd.csi.ceph.com',
+];
+
+// To check if the provisioner is OCS based
+export const isCephProvisioner = (scProvisioner: string): boolean => {
+  return cephStorageProvisioners.some((provisioner: string) =>
+    _.endsWith(scProvisioner, provisioner),
+  );
+};
+
+// To check that the storage class isn't noobaa based
+export const filterScOnProvisioner = (sc: StorageClass, provisioner: string = '') =>
+  sc.provisioner.includes(provisioner);
```

Comment:
The comment and use of this function are a little bit confusing. It might be worth making this function return the boolean value you need instead of returning the opposite, then inverting it where it's used. Just invert the return value of this function.
```suggestion
  !sc.provisioner.includes(provisioner);
```